sketcherMinimizer.h

```diff
@@ -128,7 +125,8 @@ class EXPORT_COORDGEN sketcherMinimizer
     void minimizeAll();
 
     /* assign coordinates to given molecule */
-    void minimizeMolecule(sketcherMinimizerMolecule* molecule);
+    // I don't think this function is defined
```

Comment:
When I went to add a function to call `m_miminizer.minimizeMolecule()`, it wouldn't let me name it `minimizeMolecule` since this function was already declared. However, I couldn't find an implementation or use of `sketcherMinimizer::minimizeMolecule`. 

I'm wondering if there is a way to flag functions that are declared in headers but not implemented?

Reply:
yeah this is not implemented anywhere

Reply:
we can get rid of it